content/docs/join_us/recruitment_FAQs.md

```diff
@@ -241,6 +241,26 @@ This list is not exhaustive, but some examples include:
 - Research programming support (running [drop-in sessions]({{< relref "docs/regular_events/drop-in_sessions" >}}) for people at the Turing)
 - Knowledge sharing (organising our [tech talks]({{< relref "docs/regular_events/lunchtime_tech_talks.md" >}}))
 
+### Can you tell me more about what it is like working at The Alan Turing Institute?
+
+At The Turing, we want to make sure you feel comfortable and supported in how you work. Here’s a bit about what to expect:
+
+- **Hybrid Working**: We only ask that you come into the office four days a month — the rest is up to you! Some of the team work primarily remotely, while others come in every day. There is no pressure or expectation to be in the office more often than the minimum four days.
+
+- **Flexible Hours**: Our core hours are 10 am to 4 pm, but you’re free to work whenever works best for you. Whether you’re an early bird or prefer working later, we trust you to manage your time in a way that fits your life and responsibilities. We also understand that everyone has commitments — whether it’s a doctor’s appointment, picking up kids, or running errands — so taking time for those is completely fine.
+
+- **Work-Life Balance**: We value a healthy work-life balance and believe you should be able to switch off outside of work. There is no expectation to respond to emails or messages outside your working hours!
+
+- **Hot Desks**: We use hot desks, meaning you can sit wherever you like when you’re in the office, on a first-come, first-served basis. The Research Engineering Group typically sits near one another on the first floor. If you need a permanent or bespoke workspace (such as standing/adjustable desks, footrests, or ergonomic keyboards), we can work with the facilities team to make it happen.
```

Comment:
i have had non-positive experiences trying to get little things here and there -- e.g. i'd be very surprised if IT let someone buy an ergonomic keyboard given their price points. literally have had days worth of conversations over a year on just getting a laptop with a US layout keyboard (or external one), and it basically never happened -- eventually ended up with a 30 pound one from amazon (at least they did pay for that in the end!)

would love to be proven wrong and i hope it's as you describe it here :')

Reply:
oh no, sorry to hear it :( I will limit to the desks and footrests then as I know they have been requested and approved for the office in the past.